### ISSUE TITLE
Refer to links by text

### ISSUE THREAD

**Issue by pablobm** [heynemann/pyccuracy#54]
I'm using Pyccuracy for the first time today and, coming from a Ruby/Cucumber background, I wonder why I can't refer to links by text. Instead I am forced to refer to them by id/name.

I find this surprising, as I think it defeats the point of Pyccuracy. Also, it should be very simple to implement, so I wonder whether there's a design decision behind this.

In case you think this feature would be useful, see below the code I have used to implement it, with test included:

```
diff --git a/pyccuracy/drivers/core/selenium_element_selector.py b/pyccuracy/drivers/core/selenium_element_selector.py
index de1b43a..de51c0c 100644
--- a/pyccuracy/drivers/core/selenium_element_selector.py
+++ b/pyccuracy/drivers/core/selenium_element_selector.py
@@ -60,7 +60,7 @@ class SeleniumElementSelector(object):
         Returns a regular expression that matches link(a) tags with
         the specified argument as id or name.
         '''
-        return r"//a[(@name='%s' or @id='%s')]" % (element_name, element_name)
+        return r"//a[(@name='%s' or @id='%s' or contains(., '%s'))]" % (element_name, element_name, element_name)

     @staticmethod
     def checkbox(element_name):
diff --git a/tests/acceptance/action_tests/test_click_link_en-us.acc b/tests/acceptance/action_tests/test_click_link_en-us.ac
index e4a847c..b49f8e6 100644
--- a/tests/acceptance/action_tests/test_click_link_en-us.acc
+++ b/tests/acceptance/action_tests/test_click_link_en-us.acc
@@ -17,3 +17,11 @@ When
        I click "lnkItWorks2" link and wait
 Then
        I see "It Works!" title
+
+Scenario 3 - Checking that we can click a link by text^M
+Given^M
+       I go to "link_tests.htm"^M
+When^M
+       I click "It Worked 2" link and wait^M
+Then^M
+       I see "It Works!" title
```

**Comment by heynemann** [comment]
Thanks a lot for the patch! You are absolutely right! It makes a lot of sense to click a link by it's text. We just didn't think of it! :) Your patch will definitely be applied in Pyccuracy's next release.

Cheers,
Bernardo Heynemann

**Comment by heynemann** [comment]
Fixed in #8b2e93c. Will be in next release.

**Comment by pablobm** [comment]
Awesome, thanks
